pre_push.py

```diff
@@ -37,7 +37,7 @@ def run_static():
     Otherwise, it will return statuscode 1
     """
     success = True
-    success &= do_process(["black *.py docs praw tests"], shell=True)
+    success &= do_process(["black ."], shell=True)
```

Comment:
I like the simplicity of this approach. Perhaps we can use the same exclusion as `flake8` below?

Reply:
Flake8 does not need exclusions as it uses a pre-set and .gitignore for excluding stuff.